scene/gui/text_edit.cpp

```diff
@@ -3152,7 +3152,7 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 
 	if (k.is_valid()) {
 		// Ctrl + Hover symbols
-#ifdef OSX_ENABLED
+#ifdef APPLE_STYLE_KEYS
 		if (k->get_keycode() == KEY_META) {
 #else
 		if (k->get_keycode() == KEY_CONTROL) {
```

Comment:
I wonder if we could have an OS-specific keycode to avoid this kind of `ifdef`. Something like `KEY_MASK_CMD`, but for standard keys rather than modifiers (`KEY_CMD`?).

cc @bruvzg

Reply:
Adding `KEY_CMD` define to the `core/os/keyboard.h`/`core/core_constants.cpp` should work in the same way.

```c++
// core/os/keyboard.h //
#ifdef APPLE_STYLE_KEYS
	KEY_CMD = KEY_META,
#else
	KEY_CMD = KEY_CONTROL,
#endif
```
```c++
// core/core_constants.cpp //
BIND_CORE_ENUM_CONSTANT_NO_VAL(KEY_CMD); //to avoid listing value in the docs
```

Reply:
Sounds good, but I might leave this for a future PR.

Reply:
@Calinou https://github.com/godotengine/godot/pull/43888#issuecomment-735028036

> * I added bool Input::is_modifier_pressed(int p_keycode_mask) where you can pass a mask like KEY_MASK_CMD or KEY_MASK_SHIFT and it will return whether one of the keys that make up that mask is pressed.